scripts/release.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+# should be run as:
+# ./release.sh x.x.x start|end description
+set -eo pipefail
+
+USAGE="Usage:
+${0} x.x.x start|end description, E.g. to start a release run:
+${0} 2.4.0 start Maintenance release
+then to finish the release run:
+${0} 2.4.0 end"
+
+protocol_repo=valerifilipov/repo2
+node_repo=valerifilipov/repo1
+
+#generate_post_data release description
+generate_post_data() {
+cat <<EOF
+{
+  "tag_name": "$1",
+  "target_commitish": "master",
+  "name": "$2",
+  "body": "$1",
+  "draft": false,
+  "prerelease": $prerelease
+}
+EOF
+}
+
+curl_headers=(
+    '-H' "Accept: application/vnd.github.v3+json"
+    '-H' "Authorization: token $VAULT_GITHUB_TOKEN"
```

Comment:
This shouldn't coupled to Vault. Use generic variable and make sure it's exported in the import-secrets.
Also use required variables where appropriate, like this case it should be `${VAULT_GITHUB_TOKEN:?}`

Reply:
Addressed in 886247f